README.md

```diff
@@ -390,6 +392,12 @@ Optional parameter to override the default data poll size (in minutes) of each d
 #### Queue Data Poll Interval
 Optional parameter to override the default data poll interval (in milliseconds) to poll the historical data queue.  If no value is specified in the configuration file, the value will be read from QUEUE_DATA_POLL_INTERVAL_MILLIS_DEFAULT from "src/com/hms_networks/americas/sc/thingworx/TWConnectorConsts.java".
 
+#### Payload Maximum Data Points
+Optional parameter to control the maximum number of data points which can be added to data payloads which are sent to Thingworx. If no value is specified in the configuration file, the value will be read from CONNECTOR_CONFIG_DEFAULT_PAYLOAD_MAX_DATA_POINTS in "src/com/hms_networks/americas/sc/thingworx/TWConnectorConsts.java".
+
+#### Payload Send Interval (Millis)
```

Comment:
Implementation will sleep thread. It might be more accurate to state: application will wait at a minimum this interval before sending payload.

Reply:
updated/changed